OgreMain/src/OgreDDSCodec.cpp

```diff
@@ -1043,6 +1066,17 @@ namespace {
         return BLANKSTRING;
 
     }
+    //---------------------------------------------------------------------
+    bool DDSCodec::setParameter(const String& name, const String& value)
+    {
+        if (name == "decode_enforce")
+        {
+            StringConverter::parse(value, mDecodeEnforce);
```

Comment:
```suggestion
            return StringConverter::parse(value, mDecodeEnforce);
```

in case the value string contains some garbage